restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/changelog_depreciated/mariadb/update_script_Aug07_2017.sql

```diff
@@ -0,0 +1,48 @@
+#SQL Script for MySQL/MariaDB to update DB with the schema changes
```

Comment:
I thought that we provide update scripts if DB changes. btw, @deruelle said that all changes for this feature in RC we should make in public repository.